### DISCUSSION TITLE
Experimental Servers Authentication Token Missing

### DISCUSSION THREAD

**Discussion by CrazyLegsFE** [wolveix/satisfactory-server#50] [Q&A] [answered]
I have been troubleshooting the new experimental servers not working using the normal guide. I was getting an "Authentication token missing" when trying to log in to the server.  The fix was strange as I thought all the ports were forwarded internally in the container. I had to assign ports 15000 and 15777 ports in addition to the 7777.  I hope this helps someone else as well.

V/R
CL

**Comment by wolveix** [comment, marked as answer]
This is clearly documented in the README :) You must have all three ports forwarded.

**Reply by CrazyLegsFE** [reply]
Absolutely my fault I must have read over that 10 times and didn’t actually process it. For some reason I was thinking it was done within the container even though it clearly says they have to be exposed.

**Reply by wolveix** [reply]
Haha, no problem! It's easily done :)
